### PR TITLE
Add C++ solution without template for the USACO Feb. 2020 Mad Scientist

### PR DESCRIPTION
**Pull request by BitLegion**
Add the C++ solution without any template for the 2020 February USACO Competition Bronze Problem 2, "Mad Scientist"

### CODE REVIEW

**Comment by frodakcin** [comment]
Add it to this file: https://github.com/cpinitiative/usaco-guide/blob/master/solutions/USACO%20feb20-breedflip.mdx instead of creating a new file

**Review by frodakcin — CHANGES_REQUESTED** [review]
(wrong file, read above comment)

**Comment by BitLegion** [comment]
Sorry, I just edited my fork so that the solution went in the corresponding .mdx and I deleted the unnecessary file that I had initially created

**Comment by frodakcin** [comment]
mhm no worries! Typically, the process is [here](https://usaco.guide/general/contributing#adding-a-solution). The file alr exists so we should use it.

**Review by frodakcin — CHANGES_REQUESTED** [review]
Now that there's solns in multiple languages, we can break into diff lang sections. Check https://github.com/cpinitiative/usaco-guide/blob/master/docs/Content%20Documentation.md#language-specific-content

**Comment by BitLegion** [comment]
I see, would you like me to do that?

**Comment by frodakcin** [comment]
Yes, please

**Comment by BitLegion** [comment]
I have updated to add proper language-specific content formatting. Is this fine?

**Comment by BitLegion** [comment]
I'm not sure what is going on with the tests right now; could anyone please explain?

File: solutions/USACO feb20-breedflip.mdx [hunk 1]
```diff
@@ -15,33 +60,9 @@ sys.stdout = open("breedflip.out","w")
...
 
-    return count
+<PySection/>
 
-s = [0] + s + [0]
-fout.write(str((count('01', s) + count('10', s))//2))
-```
- -->
\ No newline at end of file
+<LanguageSection/>
```
Comment (dolphingarlic) [inline solutions/USACO feb20-breedflip.mdx:68]: These tags are wrong. They should instead be like `</LanguageSection>`

**Review by dolphingarlic — CHANGES_REQUESTED** [review]

File: solutions/USACO feb20-breedflip.mdx (hunk 1 above)
Comment (BitLegion) [reply solutions/USACO feb20-breedflip.mdx]: I initially tried that, but it seems as though there is something else that was wrong. I switched it just to be safe.

**Comment by dolphingarlic** [comment]
Also what's wrong with the official solution? I don't see it using a template

**Comment by frodakcin** [comment]
Maybe try adding lines between the flags, like
```
<LanguageSection>

<CPPSection>
...
```
instead of
```
<LanguageSection>
<CPPSection>
...
```

Also, extra tabs/spaces (outside of code blocks), especially those on empty lines, will cause errors as well.

**Comment by dolphingarlic** [comment]
Sorry, but I'm going to close this PR because the official editorial already has a C++ solution without a template and I don't see the point in having another one. (Plus, this editorial isn't even linked anywhere in the guide). If anyone disagrees, feel free to reopen it

**Pull request closed without merging**